sdk/python/feast/infra/online_stores/helpers.py

```diff
@@ -53,3 +54,13 @@ def _mmh3(key: str):
     """
     key_hash = mmh3.hash(key, signed=False)
     return bytes.fromhex(struct.pack("<Q", key_hash).hex()[:8])
+
+
+def compute_entity_id(entity_key: EntityKeyProto) -> str:
+    """
+    Compute Datastore Entity id given Feast Entity Key.
+
+    Remember that Datastore Entity is a concept from the Datastore data model, that has nothing to
+    do with the Entity concept we have in Feast.
```

Comment:
Remove Datastore from the docstring.
```suggestion
    Compute Entity id given Feast Entity Key for online stores.

    Remember that Entity here refers to `EntityKeyProto` which is used in some online stores to encode the keys.
    It has nothing to do with the Entity concept we have in Feast.
```

Reply:
done